src/components/FieldLabel/index.jsx

```diff
@@ -21,7 +21,7 @@ const FieldLabel = ({
       <strong>{text}</strong>
       {helpText && id && <FieldHelp id={`${id}-help`} tip={helpText} />}
       {requireText}
-      {extraText}
+      <div><span className="text-muted">{extraText}</span></div>
```

Comment:
Eh, you *could* put the className on the div, not the span. But it's all the same.